src/std/io/file-test.ss

```diff
@@ -0,0 +1,66 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; file io test
+(import :std/test
+        :std/text/utf8
+        :std/misc/ports
+        :std/os/temporaries
+        ./interface
+        ./file
+        ./bio/api)
+(export file-io-test)
+
+(def text "the quick brown fox jumped over the fence")
```

Comment:
fence? You mean lazy dog, so it's a pangram.

Reply:
haha, i can make the fox jump over a dog, sure ;)